app/controllers/users/mfa_selection_controller.rb

```diff
@@ -9,7 +9,7 @@ class MfaSelectionController < ApplicationController
     before_action :multiple_factors_enabled?
 
     def index
-      @two_factor_options_form = TwoFactorOptionsForm.new(current_user)
+      two_factor_options_form
```

Comment:
since `two_factor_options_form` is a memoized method, we could lazily call it later and still get the same value. Is there a reason to instantiate it here?

This jumps out at me because we're calling the method but not using the result anywhere, so I think we can just remove it

Reply:
I know we call it out as a global variable to be used in the view later. I guess i can also call it as a helper method.